Source/WebCore/platform/graphics/gstreamer/WebKitWebSourceGStreamer.cpp

```diff
@@ -925,6 +925,11 @@ void StreamingClient::handleResponseReceived(const ResourceResponse& response)
         return;
     }
 
+    if(response.isRedirected()){
```

Comment:
OK this looks better. Just a spacing complaint: remember that in WebKit we leave a space after the `if` and before the `{`, like `if (response.isRedirected()) {`